cypress/support/fragments/checkin/modals/checkInClaimedReturnedItem.js

```diff
@@ -21,5 +21,9 @@ export default {
   },
   verifyModalIsClosed() {
     cy.expect(checkInModal.absent());
+  },
+  checkModalMessage: (item) => {
+    const message = `${item.title} (${item.materialType}) (Barcode: ${item.barcode}) has been claimed returned`;
+    cy.expect(Modal({ content: including(message) }).exists());
```

Comment:
please check also the enable state of three buttons
Cancel, Found by library, Returned by patron

Reply:
done